server/odc-core/src/main/resources/i18n/ErrorMessages.properties

```diff
@@ -67,6 +67,7 @@ com.oceanbase.odc.ErrorCodes.ConnectionReadonly=Access denied. The current data
 com.oceanbase.odc.ErrorCodes.ConnectionTempOnly=Access denied. Creating persistent data sources is not allowed.
 com.oceanbase.odc.ErrorCodes.ConnectionFlowConfigNotExists=No approval process found for the current data source, please contact the administrator.
 com.oceanbase.odc.ErrorCodes.ConnectionUnsupportedDBVersion=Unsupported database version: {0}.
+com.oceanbase.odc.ErrorCodes.TenantConnectionAddressNotFind=The tenant has no connectable address. Please go to the OBCloud Console to create a private/public network connection.
```

Comment:
There are no available endpoints for this tenant. Please navigate to the OceanBase Cloud console to create a private or public endpoint.

Reply:
ok～